feat: Support JSON for consuming orchestration (#346)
SAP/ai-sdk-js: packages/orchestration/README.md

return response.getContent();
```

> ### Using a JSON configuration from AI Launchpad

Reviewer: ```suggestion
### Using a JSON Configuration from AI Launchpad
```

> If you already have a orchestration workflow created in AI Launchpad, you can copy or download the configuration as a JSON and use it directly with the orchestration client.

Reviewer 2: ```suggestion
If you already have an orchestration workflow created in AI Launchpad, you can either download the configuration as a JSON file or copy the JSON string directly to use it with the orchestration client.
```

```ts
const jsonConfig = await fs.promises.readFile(
  'path/to/orchestration-config.json',
  'utf-8'
);

> // Or alternatively, you can also provide the JSON configuration as a plain string in the code directly.

Reviewer: ```suggestion
// Alternatively, you can provide the JSON configuration as a plain string in the code.
```

> // const jsonConfig = '... your JSON ...'

Reviewer 2: Please refer to the [styleguide](https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#placeholders) for placeholders

> const response = await new OrchestrationClient(jsonConfig).chatCompletion();

Reviewer 3: Like the API 👍
